docs/contributing.md

```diff
@@ -33,7 +33,7 @@ To make contributing as easy and fast as possible, you'll want to run tests and
 *pydantic* has few dependencies, doesn't require compiling and tests don't need access to databases, etc.
 Because of this, setting up and running the tests should be very simple.
 
-You'll need to have a version between **Python 3.7 and 3.11**, **virtualenv**, **git**, and **make** installed.
+You'll need to have a version between **Python 3.7 and 3.11**, **virtualenv**, **git**, **pdm** and **make** installed.
```

Comment:
I think we don't have `pdm` in `V1`

Reply:
`make install` failed without pdm being installed so I added it.

Reply:
You are right. Sorry!